test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpRazorCommentsTest/MultipleRazorCommentsInSameLineInMarkup.stree.txt

```diff
@@ -1,42 +1,33 @@
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (5:1,0) - Tokens:1
-            SyntaxKind.RazorCommentTransition;[@];
-        MetaCode span - Gen<None> - [*] - SpanEditHandler;Accepts:None - (6:1,1) - Tokens:1
-            SyntaxKind.RazorCommentStar;[*];
-        Comment span - Gen<None> - [] - SpanEditHandler;Accepts:Any - (7:1,2) - Tokens:1
-            SyntaxKind.Unknown;[];
-        MetaCode span - Gen<None> - [*] - SpanEditHandler;Accepts:None - (7:1,2) - Tokens:1
-            SyntaxKind.RazorCommentStar;[*];
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (8:1,3) - Tokens:1
-            SyntaxKind.RazorCommentTransition;[@];
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (9:1,4) - Tokens:1
-        SyntaxKind.Unknown;[];
-    Markup span - Gen<None> - [  ] - SpanEditHandler;Accepts:Any - (9:1,4) - Tokens:1
-        SyntaxKind.Whitespace;[  ];
-    Comment block - Gen<RazorComment> - 4 - (11:1,6)
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (11:1,6) - Tokens:1
-            SyntaxKind.RazorCommentTransition;[@];
-        MetaCode span - Gen<None> - [*] - SpanEditHandler;Accepts:None - (12:1,7) - Tokens:1
-            SyntaxKind.RazorCommentStar;[*];
-        Comment span - Gen<None> - [] - SpanEditHandler;Accepts:Any - (13:1,8) - Tokens:1
-            SyntaxKind.Unknown;[];
-        MetaCode span - Gen<None> - [*] - SpanEditHandler;Accepts:None - (13:1,8) - Tokens:1
-            SyntaxKind.RazorCommentStar;[*];
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (14:1,9) - Tokens:1
-            SyntaxKind.RazorCommentTransition;[@];
-    Markup span - Gen<None> - [LF] - SpanEditHandler;Accepts:Any - (15:1,10) - Tokens:1
-        SyntaxKind.NewLine;[LF];
-    Tag block - Gen<None> - 4 - (17:2,0)
-        Markup span - Gen<Markup> - [</p>] - SpanEditHandler;Accepts:Any - (17:2,0) - Tokens:4
-            SyntaxKind.OpenAngle;[<];
-            SyntaxKind.ForwardSlash;[/];
-            SyntaxKind.Text;[p];
-            SyntaxKind.CloseAngle;[>];
+RazorDocument - [0..21)::21 - [<p>LF@**@  @**@LF</p>]
+    MarkupBlock - [0..21)::21
+        MarkupTagBlock - [0..3)::3 - [<p>]
+            MarkupTextLiteral - [0..3)::3 - [<p>] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                OpenAngle;[<];
+                Text;[p];
+                CloseAngle;[>];
+        MarkupTextLiteral - [3..5)::2 - [LF] - Gen<Markup> - SpanEditHandler;Accepts:Any
+            NewLine;[LF];
+        RazorComment - [5..9)::4
+            RazorCommentTransition;[@];
+            RazorCommentStar;[*];
+            RazorCommentLiteral;[<Missing>];
+            RazorCommentStar;[*];
+            RazorCommentTransition;[@];
+        MarkupTextLiteral - [9..9)::0 - [] - Gen<Markup> - SpanEditHandler;Accepts:Any
+            Marker;[];
+        MarkupEphemeralTextLiteral - [9..11)::2 - [  ] - Gen<None> - SpanEditHandler;Accepts:Any
```

Comment:
wtf, why this gen none? Looks like a bug.

Reply:
Filed https://github.com/aspnet/Razor/issues/2726